packages/ui/src/views/layouts/SandboxLayout.tsx

```diff
@@ -10,44 +10,26 @@ interface ColumnsProps {
 
 function Root({ children }: { children: ReactNode }) {
   return (
-    <main className="h-screen" role="main">
+    <main className="grid min-h-screen grid-cols-[auto_1fr]" role="main">
```

Comment:
imo, the grid layout should be fixed at [220px 1fr] instead of auto with for first column

Reply:
This doesn't work with 220px_1fr, which should ideally work with tailwind. I still see some problem with tailwind setup. I had to setup a class for top-[100px] as well, which I would expect to work when tailwind is setup correctly without having to define a class for that. I'll discuss it with Kelvin later. For now, we can skip this.